faunadb/client.go

```diff
@@ -35,11 +35,22 @@ func HTTP(http *http.Client) ClientConfig { return func(cli *FaunaClient) { cli.
 EnableTxnTimePassthrough configures the FaunaClient to keep track of the last seen transaction time.
 The last seen transaction time is used to avoid reading stale data from outdated replicas when
 reading and writing from different nodes at the same time.
+
+Deprecated: This function is deprecated since this feature is enabled by default.
 */
 func EnableTxnTimePassthrough() ClientConfig {
 	return func(cli *FaunaClient) { cli.isTxnTimeEnabled = true }
 }
 
+/*
+DisableTxnTimePassthrough configures the FaunaClient to not keep track of the last seen transaction time.
+The last seen transaction time is used to avoid reading stale data from outdated replicas when
+reading and writing from different nodes at the same time.
```

Comment:
Could you mention here that this is dangerous, it can lead to data inconsistencies, and not recommended? Something like "use at your own risk"...